docker-compose.yml

```diff
@@ -28,6 +28,11 @@ services:
     volumes:
       - fsdata:/data
 
+  sass:
+    image: larryprice/sass
+    volumes:
+      - rgd/core/static/css/sass:rgd/core/static/css/
```

Comment:
I think this needs to mount to `:/src` for the `larryprice/sass` image to compile it

I'm testing locally.........